Makefile

```diff
@@ -1,26 +1,36 @@
 ifndef LIGO
-LIGO=docker run --rm -v "$(PWD)":"$(PWD)" -w "$(PWD)" ligolang/ligo:next
+LIGO=docker run --platform linux/amd64 --rm -v "$(PWD)":"$(PWD)" -w "$(PWD)" ligolang/ligo:0.43.0
 endif
 
-json=--michelson-format json
+PROTOCOL_OPT=--protocol ithaca
```

Comment:
bot sure this is needed, we add this flag when testing specific features https://ligolang.org/docs/protocol/ithaca